packages/core/template/project/_markbind/variables.json

```diff
@@ -0,0 +1,3 @@
+{
+    "jsonVariableExample": "Your variables can be defined here as well"
```

Comment:
Is this used anywhere? I think if we are going to put example variables in, lets have it used in the site.